src/Info.ts

```diff
@@ -0,0 +1,107 @@
+import { fileBackedObject } from "./FileBackedObject";
+import { SharedSettings } from "./SharedSettings";
+
+import Discord = require("discord.js");
+
+// If the Collection contains anything in the other array
+const findOne = (arr1: Discord.Collection<string, Discord.Role>, arr2: Array<any>) => {
+    return arr1.some(v => {
+        return arr2.indexOf(v) >= 0;
+    });
+};
+
+export interface InfoData {
+    Command: string;
+    Message: string;
```

Comment:
My only issue is that these fields are capitalized. TypeScript conventions use lowerCamelCase for field names. I guess that config values elsewhere also use PascalCase, but that's because Q is a goof.

Reply:
Yeah, I didn't like capitalizing either, but I was just doing it for consistency. I think we either need to keep it like it is and commit to refactoring all of them later or going back right now and changing all of them.